### ISSUE TITLE
[A11y]

### ISSUE THREAD

**Issue by abdinasirjamal171** [IBMa/equal-access#406]
## Environment

> Operating system:

> Browser:

> Automated testing tool and ruleset:

> Assistive technology used to verify:

## Detailed description

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

## Additional information

- Screenshots or code
- Notes

**Comment by abdinasirjamal171** [comment]
Kindly help in payment process

**Comment by tombrunet** [comment]
You appear to have the wrong project. There are no payments involved with this project.
